templates/index.html

```diff
@@ -1,8 +1,108 @@
-<title>Community website</title>
-<body>
-<ul>
-    <li><a href="/gci/">Google Code-in</a>
-    <li><a href="/activity/">GitHub activity</a>
-    <li><a href="/twitter/">Twitter Feed</a>
-</ul>
+{% extends 'includes.html' %}
+
+{% block content %}
+    <div class="section no-pad-bot" id="index-banner">
+        <div class="container">
+            <h1 class="header center  col-test">Welcome</h1>
+            <div class="row center">
+                <h5 class="header col s12 light">Hello, world. You are at the {{ org_name }} community website.</h5>
+                <h4 class="header col s12 light">Meet awesome mentors</h4>
+
+            </div>
+            <div class="row">
+
+                <div class="col s4">
+                    <div class="card" style="overflow: visible;">
+                        <div class="card-image waves-effect waves-block waves-light">
+                            <img class="activator"
+                                 src="https://api.{{ org_name }}.io/en/latest/_static/images/{{ org_name }}_logo.svg">
```

Comment:
No, no, no.  This is coala specific URLs.

What is more, we have actually already built a solution for generic logos.  You only need to use it.

Reply:
can you please tell me the usage? I was coming up with a proposal of introducing a config file to have all the org related information in my next PR.

Reply:
@jayvdb should I use the `header.png` file?

Reply:
No it shouldnt use `header.png` .  I've created https://github.com/coala/community/issues/55 to remove that.

Reply:
Please stop consuming so much of other mentors time.  It is counter productive.  search this repo for "logo".

Reply:
If your next iteration of this PR does not comply with the goals of this project, I am closing the PR and unassigning.